bench/parse.rs

```diff
@@ -20,7 +20,7 @@ pub fn parse_benchmark(c: &mut Criterion) {
     group.bench_function("ada_url", |b| {
         b.iter(|| {
             URLS.iter().for_each(|url| {
-                black_box(url).parse::<ada_url::Url>().unwrap();
+                ada_url::Url::try_from(*black_box(url)).unwrap();
             })
         })
     });
```

Comment:
@anonrig Is it necessary to run benchmark in a mere binding library?

Reply:
> Is it necessary to run benchmark in a mere binding library?

I am not sure what you mean, but if you are asking whether github.com/ada-url/rust should be benchmarked, I would say "yes, definitively". How the binding is done can have significant impact on the performance. It is important to measure it.

Reply:
I agree with @lemire